### ISSUE TITLE
[low] JAWS+Chrome: Many Pages: Visually grouped lists of contacts are not programmatic lists

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5650]
Prerequisites:
- User has created personal account and is logged into same.
- The user has anywhere approximately 15+ contacts and several started group chats in recent history panel
on the left side of the page.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Click on "+" button and select "New Group" option.
3. Select several contacts (3+) to add to the new group by focusing on the contact with the tab key and pressing Enter. Each contact shows a green checkmark and is paced into a list just under the search field.
4. Tab/Shift+Tab through the contacts in the panel and note what is announced when the focus enters or exits and of the displayed lists. (i.e. selected, RECENTS, CONTACTS).

## Expected Result:
The contacts are split into several visual lists that show the user at a glance what has been selected, used recently, and all the rest of the saves contacts in addition to how many items are in each list. The visual list structure should be programmatic such that the the user is notified when they have entered or exited a list and the number of items in the list.

## Actual Result:
There is no list structure for the different visual lists of contacts. No list information is announced when the focus moves through the lists.

HTML:
`<div class="css-1dbjc4n"><div class="css-1dbjc4n"><div class="css-1dbjc4n"></div></div><div class="css-1dbjc4n"><div class="css-1dbjc4n"><div tabindex="0" class="css-1dbjc4n r-1awozwy r-1loqt21 r-1otgn73 r-1i6wzkk r-lrvibr" style="background-color: rgb(255, 255, 255); flex-direction: row; justify-content: space-between; padding-left: 20px; padding-right: 20px; text-decoration-line: none; transition-duration: 0s; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -webkit-box-pack: justify;"><div class="css-1dbjc4n r-13awgt0 r-16y2uox r-1777fci" style="height: 64px; min-width: 0px; padding-bottom: 12px; padding-top: 12px; text-decoration-line: none;"><div class="css-1dbjc4n r-1awozwy" style="flex-direction: row; -webkit-box-orient: horizontal; -webkit-box-direction: normal;"><div class="css-1dbjc4n" style="height: 40px; margin-right: 16px; width: 40px;"><div class="css-1dbjc4n r-633pao"><div class="css-1dbjc4n r-1mlwlqe r-1udh08x r-417010" style="border-radius: 40px; height: 40px; width: 40px;"><div class="css-1dbjc4n r-1niwhzg r-vvn4in r-u6sd8q r-4gszlv r-1p0dtai r-1pi2tsx r-1d2f490 r-u8s1d r-zchlnj r-ipm5af r-13qz1uu r-1wyyakw" style="background-image: url(&quot;https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/avatar_7.png&quot;
);"></div>...</div>`

Other occurrences:
- Chat page: The list of chat sessions is not a programmatic list.

## Workaround:
No workaround

## Area issue was found in:
Create new group overlay, Chat page

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users will not get the same information announced to them that the visual list structure and placement provides sighted users. They will not be able to easily tell they have navigated into or out of the lists or how many items to expect in the list without navigating through and counting each one.

## Suggested resolution:
Use non-ordered lists <ul> to implement each separate list in the panel as a programmatic list where each contact row is a list item <li>. Note: Do not include the list headings as list items in the list.
https://www.w3.org/WAI/WCAG21/Techniques/html/H48.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5185078_Chrome_Jaws-Chat-Visual_list_of_chat_session_is_not_programmatic](https://user-images.githubusercontent.com/88733897/136042896-ae916564-1dc9-49f8-9148-c5f0563fe10c.png)

https://user-images.githubusercontent.com/88733897/136042908-a6ee01d0-b292-4ada-933a-92d8e2656089.mp4

![Bug5185078_Chrome_Jaws-Create_New_Group-Visually_grouped_lists_of_contacts_are_not_programmatic_lists](https://user-images.githubusercontent.com/88733897/136042921-d60dd3a6-adb1-43ea-9dae-7aaacc65cc33.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
